creator-node/src/TranscodingQueue.js

```diff
@@ -157,7 +157,7 @@ class TranscodingQueue {
    * @param {string} fileDir
    * @param {string} fileName
    * @param {Object} logContext to create a logger.child(logContext) from
-   * @returns {Object} response in the structure 
```

Comment:
Because typescript tries to type check with `Object` and it's a loaded word. This file is on my list to get typed soon so my thought was that I'd properly type it then